app-text/qpdfview/patches/qpdfview-0.4.18.patchset

```diff
@@ -42,3 +42,156 @@ index 6363c32..dc10a8c 100644
++isEmpty(TARGET_INSTALL_PATH):TARGET_INSTALL_PATH = $appsDir/qPDFView
++isEmpty(PLUGIN_INSTALL_PATH):PLUGIN_INSTALL_PATH = $appsDir/qPDFView/lib
++isEmpty(DATA_INSTALL_PATH):DATA_INSTALL_PATH = $appsDir/qPDFView/data
++isEmpty(MANUAL_INSTALL_PATH):MANUAL_INSTALL_PATH = $appsDir/qPDFView/man/man1
++isEmpty(ICON_INSTALL_PATH):ICON_INSTALL_PATH = $appsDir/qPDFView/icons/hicolor/scalable/apps
++isEmpty(LAUNCHER_INSTALL_PATH):LAUNCHER_INSTALL_PATH = $appsDir/qPDFView/applications
++isEmpty(APPDATA_INSTALL_PATH):APPDATA_INSTALL_PATH = $appsDir/qPDFView/appdata
+ 
+ win32:include(qpdfview_win32.pri)
+ os2:include(qpdfview_os2.pri)
+-- 
+2.30.2
+
+
+From b5af672c965a1febb3521ed1875d712804fc6350 Mon Sep 17 00:00:00 2001
+From: FuRuYa7 <mymail@mydomain.org>
+Date: Fri, 29 Oct 2021 15:04:08 +0900
+Subject: [PATCH 2/2] Changed to reflect Japanese translation files
+
+---
+ qpdfview.pro     | 14 +++++++++++++-
+ translations.qrc | 12 ++++++++++++
+ 2 files changed, 25 insertions(+), 1 deletion(-)
+
+diff --git a/qpdfview.pro b/qpdfview.pro
+index b9c112f..2170492 100644
+--- a/qpdfview.pro
++++ b/qpdfview.pro
+@@ -41,11 +41,13 @@ TRANSLATIONS += \
+     translations/qpdfview_da.ts \
+     translations/qpdfview_de.ts \
+     translations/qpdfview_el.ts \
++    translations/qpdfview_en_AU.ts \
+     translations/qpdfview_en_GB.ts \
+     translations/qpdfview_eo.ts \
+     translations/qpdfview_es.ts \
+     translations/qpdfview_eu.ts \
+     translations/qpdfview_fi.ts \
++    translations/qpdfview_fa.ts \
+     translations/qpdfview_fr.ts \
+     translations/qpdfview_gl.ts \
+     translations/qpdfview_he.ts \
+@@ -53,21 +55,31 @@ TRANSLATIONS += \
+     translations/qpdfview_hu.ts \
+     translations/qpdfview_id.ts \
+     translations/qpdfview_it.ts \
++    translations/qpdfview_ja.ts \
+     translations/qpdfview_kk.ts \
++    translations/qpdfview_ko.ts \
++    translations/qpdfview_ku.ts \
+     translations/qpdfview_ky.ts \
+     translations/qpdfview_lt.ts \
++    translations/qpdfview_lv.ts \
+     translations/qpdfview_ms.ts \
+     translations/qpdfview_my.ts \
++    translations/qpdfview_nb.ts \
++    translations/qpdfview_nds.ts \
++    translations/qpdfview_oc.ts \
+     translations/qpdfview_pl.ts \
+     translations/qpdfview_pt.ts \
+     translations/qpdfview_pt_BR.ts \
+     translations/qpdfview_ro.ts \
+     translations/qpdfview_ru.ts \
+     translations/qpdfview_sk.ts \
++    translations/qpdfview_sr.ts \
+     translations/qpdfview_sv.ts \
+     translations/qpdfview_th.ts \
+     translations/qpdfview_tr.ts \
+     translations/qpdfview_ug.ts \
+     translations/qpdfview_uk.ts \
++    translations/qpdfview_uz.ts \
+     translations/qpdfview_vi.ts \
+-    translations/qpdfview_zh_CN.ts
++    translations/qpdfview_zh_CN.ts \
++    translations/qpdfview_zh_TW.ts
+diff --git a/translations.qrc b/translations.qrc
+index 7627a38..d631e13 100644
+--- a/translations.qrc
++++ b/translations.qrc
```

Comment:
Is this file generated? I don't see this change in upstream repository. If it's needed, it might be a good idea to send this part upstream.
see for instance https://bugs.launchpad.net/qpdfview/+bug/1841048

Reply:
Thanks